portfolio/src/test/java/com/google/sps/TestSentiment.java

```diff
@@ -0,0 +1,295 @@
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import static org.mockito.Mockito.*;
+
+import com.google.sps.functionality.SentimentAnalyzer;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.AnalyzeSentimentResponse;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.ClassifyTextRequest;
+import com.google.cloud.language.v1.ClassifyTextResponse;
+import com.google.cloud.language.v1.ClassificationCategory;
+import com.google.api.gax.rpc.InvalidArgumentException;
+import java.lang.Throwable;
+import com.google.api.gax.grpc.GrpcStatusCode;
+import io.grpc.Status;
+
+@RunWith(JUnit4.class)
+public final class TestSentiment {
```

Comment:
nit: usually each XX.java class has a unittest named as XXTest.java.

Reply:
Thanks! The file is now named "SentimentTest"